usage/index.rst

```diff
@@ -0,0 +1,11 @@
+Usage
```

Comment:
done in the faq section https://github.com/quarkslab/irma/blob/master/faq/swagger.rst